content/pages/beta-enrollment/personalization.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Personalization Beta Enrollment
+layout: page-react.html
+entryname: beta-enrollment
+---
+<div id="main">
+  <nav class="va-nav-breadcrumbs">
+    <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
+      <li><a href="/">Home</a></li>
+    </ul>
+  </nav>
+
+<div class="row-padded">
+
+# Personalization Beta Enrollment
+
+You are attempting to access the Personalization features of Vets.gov. These features are currently in active development and are therefore prone to change.
+
+Enrolling in this beta will enable the following features:
+
+- Personalized Dashboard
+- Personalized Profile
+- Account Information
+
+<div id="react-root"></div>
+</div>
```

Comment:
The react-root here will ultimately just render a button for enrolling in a beta.